dev-lua/lua-argparse/lua-argparse-0.6.0.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit toolchain-funcs
+
+DESCRIPTION="Command line argument parser for the Lua Programming Language"
+HOMEPAGE="https://github.com/mpeterv/argparse"
+SRC_URI="https://github.com/mpeterv/argparse/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc luajit"
+
+RDEPEND="
+	>=dev-lang/lua-5.1:=
+	luajit? ( dev-lang/luajit:2 )
+"
+DEPEND="${RDEPEND}"
```

Comment:
since you use pkgconfig below you are missing `virtual/pkgconfig` in `BDEPEND`

Reply:
fixed